utils/clean_up_repo.py

```diff
@@ -0,0 +1,61 @@
+"""
+The Qxf2 automation repository ships with example tests.
+Run this file to delete all the example files and start fresh with your example.
+After this script runs, you will need to edit a few files to configure them to suit your own repository
+Usage: python clean_up_repo.py
+"""
+import os
+REPO_DIR = os.path.dirname(os.path.dirname(__file__))
```

Comment:
Can all this be moved into a conf file?

Ideally you will have a list of (dir,file list) that you use in `delete_files_in_dir`